source/custom/nidcpower_service.custom.cpp

```diff
@@ -1,4 +1,4 @@
-#include <nidcpower/nidcpower_service.h>
+#include <nidcpower_service.h>
```

Comment:
The custom cpps are inside the driver library and now just reference by filename (or not at all if they're empty)